WalletWasabi.Tests/UnitTests/ViewModels/SuggestionLabelsViewModelTests.cs

```diff
@@ -0,0 +1,134 @@
+using System.Collections.Generic;
+using DynamicData;
+using FluentAssertions;
+using NBitcoin;
+using WalletWasabi.Blockchain.Transactions;
+using WalletWasabi.Fluent.Models.Wallets;
+using WalletWasabi.Fluent.ViewModels.Wallets.Labels;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.ViewModels;
+
+public class SuggestionLabelsViewModelTests
+{
+	[InlineData(1, 1)]
+	[InlineData(2, 2)]
+	[InlineData(3, 3)]
+	[InlineData(100, 5)]
+	[Theory]
+	public void Given_max_top_suggestions_the_suggestion_count_should_match(int maxSuggestions, int expectedSuggestionsCount)
```

Comment:
@molnard 

As far as I can see Wasabi uses camel case for the name of tests, but the most common naming convention is this.
Should we stick to the camel case?

Reply:
Please maintain consistency. This method name is not OK.